packages/framework/src/errors/index.ts

```diff
@@ -7,3 +8,4 @@ export * from './resource.errors';
 export * from './signature.errors';
 export * from './step.errors';
 export * from './workflow.errors';
+export { isFrameworkError } from './guard.errors';
```

Comment:
We want to expose `isFrameworkError` for Bridge API clients to handle their errors neatly with error determination handled by the Framework package.